src/main/java/slimeknights/tconstruct/common/TinkerOredict.java

```diff
@@ -128,6 +148,14 @@ private static void registerCommon() {
     // Ores
     oredict(oreCobalt, "oreCobalt");
     oredict(oreArdite, "oreArdite");
+    
+    // glass
+    oredict(blockClearGlass, "blockGlass"); // no blockGlassColorless as then it is assumed as available for staining
+                                            // which blocks our own staining recipes
+    oredict(blockClearStainedGlass, "blockGlass");
+    for(int i = 0; i > 16; i++) {
+      oredict(blockClearStainedGlass, i, "blockGlass" + dyes[i]);
```

Comment:
Each one actually uses a different name, for example "blockGlassRed". I do wildcard it above for the generic glass oredict